### ISSUE TITLE
[Bug] service account authentication failure for crawling urls #2569

### ISSUE THREAD

**Issue by v-viyada** [microsoft/accessibility-insights-action#2182]
## Describe the bug

accessibility insights scan is failing to scan url using service account authentication and giving error message Authentication failed. Authentication requires a non-people service account. To learn how to set up a service account, visit: https://aka.ms/AI-action-auth. TargetCloseError: Waiting for selector #FormsAuthentication

This is used to run fine earlier with same service account and there is no change in service account password as well.

**To Reproduce**

1. Add accessibility insights task to pipeline
2. Set up the task parameters to use service account and password
3. Run the pipeline and check logs

## Expected behavior

Service account authentication should succeed and scan the URL

## Context (please complete the following information)

- OS Name & Version: 
- Azure DevOps Extension Version & Environment: 
- Browser Version: 
- Target Page: 

## Additional context

The error is coming from accessibility-insights-scan package. raised issue https://github.com/microsoft/accessibility-insights-service/issues/2569. Once it is fixed, we need to update scan package.
